src/kernels/batchnorm_functions.h

```diff
@@ -159,6 +168,10 @@
 #define MIO_BN_GFX103X 0
 #endif
 
+#ifndef MIO_BN_GFX110X
+#define MIO_BN_GFX110X 0
+#endif
+
```

Comment:
[Recommendation] Move this + lines 167-169 somewhere above lines 134-138 and then remove `defined(MIO_BN_GFX103X)` and `defined(MIO_BN_GFX110X)` from line 135. Up to you.

Reply:
I will keep it as it is now. thank you.